backend/src/EntityExtractor.Contracts/FileStoringDTO.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace eKultura.EntityExtractor.Contracts;
+
+public class FileStoringDTO
```

Comment:
I suggest naming this `DocumentFile` or RawDocument something like that. We already have classes like TextDocument and PdfDocument, so let's stay consistent.